contribute/style-guides/backend.md

```diff
@@ -75,3 +75,15 @@ Valid reasons to use a pointer include (but not necessarily limited to):
   allocating heap memory)
 * You might *need* `nil` to tell if a variable isn't set, although usually it's better to use the type's zero
   value to tell instead
+
+## Database
+
+### Foreign keys
+
+While they can be useful, we don't generally use foreign key constraints in Grafana, for historical and
+technical reasons. See this [comment](https://github.com/grafana/grafana/issues/3269#issuecomment-383328548) by Torkel 
+for context.
```

Comment:
While they can be useful, for historical and
technical reasons we do not generally use foreign-key constraints in Grafana. For context, refer to this [comment](https://github.com/grafana/grafana/issues/3269#issuecomment-383328548).

Is there a reason any specific person needs to be named explicitly (in this case Torkel)?

Reply:
@osg-grafana It's useful for us working on the code to know who is behind this architectural decision, and how it came about.